src/containers/now-playing/NowPlaying.tsx

```diff
@@ -300,6 +311,17 @@ const NowPlaying = g(
       }
     }
 
+    const onNext = () => {
+      if (track.genre === Genre.PODCASTS) {
+        const newPosition = timing.position + SKIP_DURATION_SEC
+        if (newPosition < timing.duration) {
```

Comment:
here too, should we use seek(Math.min(newPosition, timing.duration)) and not do this check